tests/acceptance/features/webUITrashbin/trashbinFilter.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Filter files/folders
+
+  As a user
+  I would like to filter files/folders
+  So that I can make the file/folder list more clear
+
+  Background:
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    And the user has created file ".simpleHiddenFile"
+    And the user has created folder ".simpleHiddenFolder"
+    And the following files have been deleted by user "user1"
```

Comment:
So basically there is only one of file, folder, hiddenFile, hiddenfolder each in the trashbin. But if you want to test the filter feature it would be better to add more of the elements of each kind so that we know that the feature actually work.

Reply:
fixed